client/src/test/java/io/pravega/client/control/impl/ControllerImplTest.java

```diff
@@ -955,6 +957,71 @@ public void getCurrentSegmentsKeyValueTable(Controller.KeyValueTableInfo request
                     responseObserver.onError(Status.INTERNAL.withDescription("Server error").asRuntimeException());
                 }
             }
+
+            @Override
+            public void listKeyValueTablesInScope(Controller.KVTablesInScopeRequest request,
+                                                  StreamObserver<Controller.KVTablesInScopeResponse> responseObserver) {
+                if (request.getScope().getScope().equals(NON_EXISTENT)) {
+                    responseObserver.onNext(Controller.KVTablesInScopeResponse
+                            .newBuilder().setStatus(Controller.KVTablesInScopeResponse.Status.SCOPE_NOT_FOUND)
+                            .build());
+                    responseObserver.onCompleted();
+                } else if (request.getScope().getScope().equals("deadline")) {
+                    // dont send any response
+                    System.err.println("i am here");
```

Comment:
Please, remove think kind of console prints from the code.

Reply:
removed